django_basin3d/apps.py

```diff
@@ -3,6 +3,12 @@
 
 from django_basin3d.catalog import load_data_sources
 
+from basin3d.core import monitor
+
+monitor.configure(loggers={'basin3d': {'level': 'DEBUG', 'handlers': ['console']},
+                           'django_basin3d': {'level': 'DEBUG', 'handlers': ['console']}})
```

Comment:
We need to move this to  settings.py.  That way folks can override/extend the logging configuration options if need be.  We are changing they how logging is configured in Django (https://docs.djangoproject.com/en/4.2/topics/logging/) and should make sure the comments in the settings.py reflect that.  Also we may want to note that in the documentation as well.
```suggestion
```

Reply:
Thanks for catching that, Val.
As we discussed, we need a bit more work to utilize the basin3d custom logger. So changed to straight up logging for now.